src/components/course/data/tests/hooks.test.jsx

```diff
@@ -1457,73 +1232,86 @@ describe('useMinimalCourseMetadata', () => {
+
+  const baseCourseMetadataValue = {
+    organization: {
+      name: mockOrgName,
+      logoImgUrl: mockLogoImageUrl,
+      marketingUrl: mockOrgMarketingUrl,
+    },
+    title: mockCourseTitle,
+    startDate: mockCourseRunStartDate,
+    duration: `${mockWeeksToComplete} Weeks`,
+    priceDetails: {
+      price: mockListPrice,
+      currency: mockCurrency,
     },
-    coursePrice: { list: mockListPrice, discount: 0 },
-    currency: mockCurrency,
   };
-  const Wrapper = ({
-    courseContextValue = baseCourseContextValue,
-    children,
-  }) => (
-    <CourseContext.Provider value={courseContextValue}>
-      {children}
-    </CourseContext.Provider>
+
+  const Wrapper = ({ children }) => (
+    <BrowserRouter>
+      <AppContext.Provider value={mockAuthenticatedUser}>
+        {children}
+      </AppContext.Provider>
+    </BrowserRouter>
   );
 
+  beforeEach(() => {
+    jest.clearAllMocks();
+    useEnterpriseCustomer.mockReturnValue({ data: mockEnterpriseCustomer });
+    useCourseMetadata.mockReturnValue(baseCourseMetadataValue);
+    useParams.mockReturnValue({ courseKey: 'test-course-key' });
+    useRedeemablePolicies.mockReturnValue({
+      data: {
+        redeemablePolicies: [],
+      },
+    });
+    useCourseRedemptionEligibility.mockReturnValue({ data: {} });
+    useSubscriptions.mockReturnValue({
+      data: {
+        customerAgreement: undefined,
+        subscriptionLicense: undefined,
+        subscriptionPlan: undefined,
+        shouldShowActivationSuccessMessage: false,
+      },
+    });
+    useEnterpriseCustomerContainsContent.mockReturnValue({
+      data: {
+        containsContentItems: false,
+        catalogList: [],
+      },
+    });
+    useEnterpriseOffers.mockReturnValue({
+      data: {
+        enterpriseOffers: [],
+        canEnrollWithEnterpriseOffers: false,
+      },
+    });
+    useCouponCodes.mockReturnValue({
+      data: {
+        couponCodeAssignments: [],
+      },
+    });
+  });
+
   it('should return the correct base course metadata', () => {
     const { result } = renderHook(() => useMinimalCourseMetadata(), { wrapper: Wrapper });
-    expect(result.current).toEqual(
-      expect.objectContaining({
-        organization: {
-          name: mockOrgName,
-          logoImgUrl: mockLogoImageUrl,
-          marketingUrl: mockOrgMarketingUrl,
-        },
-        title: mockCourseTitle,
-        startDate: mockCourseRunStartDate,
-        duration: `${mockWeeksToComplete} Weeks`,
-        priceDetails: {
-          price: mockListPrice,
-          currency: mockCurrency,
-        },
-      }),
-    );
+    expect(result.current).toEqual(baseCourseMetadataValue);
   });
 
   it('should handle empty activeCourseRun', () => {
-    const args = {
-      ...baseCourseContextValue,
-      state: {
-        ...baseCourseContextValue.state,
-        activeCourseRun: undefined,
-      },
+    const updatedCourseMetadataValue = {
+      ...baseCourseMetadataValue,
+      duration: '-',
```

Comment:
[nit/clarification] What is the `-` representing? Is this a legit possible value?

Reply:
It is a possible value for the duration value if no active course runs exist. Bringing our async conversation forward here, I am opting to leave in the specific conditional that results in this return value of `-`, reason being as a sane fallback for the empty course run state for testing purposes. It is only used in one component and is gated higher up in the component tree to avoid the user actually running into a `-` state for the course duration.